### PR TITLE
Creates Autism and ADHD PAGE

### PR DESCRIPTION
**Pull request by seemavora**
image
image

Also links pages to Navbar!

### CODE REVIEW

**Comment by seemavora** [comment]
image
image
New Design

**Review by ThaiQ — APPROVED** [review]

**Review by Annabel1054 — APPROVED** [review]

File: src/pages/Disabilities/ADHD/adhd.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import Navbar from "../../../components/navbar/navbar.js";
+import GameLayout from '../Autism/gameLayout.js'
+import './adhd.css'
+
```
Comment (seemavora) [inline src/pages/Disabilities/ADHD/adhd.js]: i removed these blank lines, ill push after everyone reviews

File: src/pages/Disabilities/Autism/autism.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import Navbar from "../../../components/navbar/navbar.js";
+import GameLayout from '../Autism/gameLayout.js'
+import './disabilities.css'
+
+import abc from './pictures/abc.png'
+import matching from './pictures/matching.jpg'
+import phonics from './pictures/phonics.jpg'
+
+
```
Comment (seemavora) [inline src/pages/Disabilities/Autism/autism.js]: same thing as above

File: src/pages/Disabilities/Autism/disabilities.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  background-color: rgba(95, 182, 176, 0.938) !important;
+  height: auto;
+  opacity: 0.8;
+  width: 100%;
+}
+
+.banner-texts h1 {
+  font-size: 30px;
+  font-weight: bold;
+  /* color: rgb(4, 35, 134); */
```
Comment (seemavora) [inline src/pages/Disabilities/Autism/disabilities.css]: removed

**Review by nanarboursal — APPROVED** [review]
looks good!

File: src/pages/Disabilities/Autism/gameLayout.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+.card-wrapper {
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  flex-wrap: wrap;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  text-align: center;
+  min-width: 400px;
+}
```
Comment (seemavora) [inline src/pages/Disabilities/Autism/gameLayout.css]: added

**Pull request merged**